packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/wdm-dto.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
this file can mostly be ignored, as it's a duplicate of an approved file from a previous PR to allow it's usage in integration tests.